ladybug_geometry/dictutil.py

```diff
@@ -0,0 +1,67 @@
+# coding=utf-8
+"""Utilities to convert any Ladybug Geometry dictionary to Python objects.
+
+Note that importing this module will import almost all modules within the
+Ladybug_geometry library in order to be able to re-serialize almost any 
+dictionary produced from the library.
+"""
+
+from ladybug_geometry.geometry2d import (Vector2D, Point2D, Ray2D, 
+                            LineSegment2D, Arc2D, Polyline2D, Polygon2D, Mesh2D)
+from ladybug_geometry.geometry3d import (Vector3D, Point3D, Ray3D,
+                            LineSegment3D, Arc3D, Polyline3D, Polyface3D, Mesh3D,
+                            Plane, Face3D, Sphere, Cone, Cylinder)
+
+def geometry_dict_to_object(ladybug_geom_dict, raise_exception=True):
+    # type (dict, bool) -> object | None
+    """
+    Args:
+        ladybug_geom_dict (dict): A dictionary of any Ladybug Geometry object. Note
+            that this should be a non-abridged dictionary to be valid.
+        raise_exception (bool): Boolean to note whether an exception should be raised
+            if the object is not identified as a part of honeybee.
+            Default: True.
```

Comment:
Ladybug_geonetry has no knowledge of honeybee. So it should be "part of ladybug_geometry"

Reply:
confirmed.